src/components/Thumbnail.js

```diff
@@ -30,7 +30,7 @@ function Thumbnail ({ index, src, thumbnail, active, onClick, type, custom, cust
 			style={{
 				backgroundImage: customContent ? '' : 'url("' + url + '")',
 				height: theme.thumbnail.height,
-				margin: `0 ${theme.thumbnail.gutter}px`,
+				margin: `0px ${theme.thumbnail.gutter}px`,
```

Comment:
somehow this line of code was getting into the browser as:
`margin: '${theme.thumbnail.gutter}px'`

don't ask me why :)